docs/tasks/federation/set-up-cluster-federation-kubefed.md

```diff
@@ -24,52 +24,47 @@ This guide assumes that you have a running Kubernetes cluster. Please
 see one of the [getting started](/docs/setup/) guides
 for installation instructions for your platform.
 
-
 ## Getting `kubefed`
 
-Download the client tarball corresponding to the latest release and
-extract the binaries in the tarball with the commands:
+Download the client tarball corresponding to the particular release and 
+extract the binaries in the tarball:
+
+### For k8s versions 1.8.x and earlier:
 
 ```shell
-# Linux
-curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/kubernetes-client-linux-amd64.tar.gz
+curl -LO https://storage.googleapis.com/kubernetes-release/release/${RELEASE-VERSION}/kubernetes-client-linux-amd64.tar.gz
 tar -xzvf kubernetes-client-linux-amd64.tar.gz
-
-# OS X
-curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/kubernetes-client-darwin-amd64.tar.gz
-tar -xzvf kubernetes-client-darwin-amd64.tar.gz
-
-# Windows
-curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/kubernetes-client-windows-amd64.tar.gz
-tar -xzvf kubernetes-client-windows-amd64.tar.gz
 ```
+> Note that the variable `RELEASE-VERSION` should be either appropriately 
+set to or replaced with the actual version needed. 
 
-> Note: The URLs in the curl commands above download the binaries for
-`amd64`. If you are on a different architecture, please use a URL
-appropriate for your architecture. You can find the list of available
-binaries on the
-[release page](https://git.k8s.io/kubernetes/CHANGELOG.md#client-binaries-1).
-
-Copy the extracted binaries to one of the directories in your `$PATH`
-and set the executable permission on those binaries.
-
+Copy the extracted binary to one of the directories in your `$PATH`
+and set the executable permission on the binary.
 
 ```shell
 sudo cp kubernetes/client/bin/kubefed /usr/local/bin
 sudo chmod +x /usr/local/bin/kubefed
-sudo cp kubernetes/client/bin/kubectl /usr/local/bin
-sudo chmod +x /usr/local/bin/kubectl
 ```
 
-### Install with snap on Ubuntu
+### For k8s versions 1.9.x:
```

Comment:
For k8s versions 1.9.x and above?

Do you plan to edit this file for every release (1.10 is around the corner)